client/src/components/base/ProjectForm.tsx

```diff
@@ -176,9 +179,23 @@ function ProjectForm({
           changeHandler={handleInput}
         />
         {!formValidation.valid && submitted && (
-          <p className="text-danger-text w-full text-center font-semibold my-2">
-            {formValidation.message}
-          </p>
+          <div
+            className="p-4 text-red-700 border rounded border-red-900/10 bg-red-50 mt-8"
+            role="alert"
+          >
+            <strong className="text-sm font-medium">
+              There {formValidation.errorCount === 1 ? "was" : "were"}{" "}
+              {formValidation.errorCount} error
```

Comment:
shall we also put the ending s for plural in case?

Reply:
Yep, nice catch, I did no pluralise "error(s)"